jvm/hermes/BUILD

```diff
@@ -36,11 +36,11 @@ merge_jni_into_android_library(
     name = "hermes-android",
     android_library = ":hermes-android-lib",
     cc_libs = [
-        "@fbjni_artifact//:libc++_shared",
-        "@fbjni_artifact//:libfbjni",
+        "//jvm/hermes/libs:libc++_shared",
+        "//jvm/hermes/libs:libfbjni",
         "//jvm/hermes/libs:libjsi",
         "//jvm/hermes/src/main/jni:hermes_jni_lib",
-        "@rn_hermes_android//:libhermes",
+        "//third_party/rn:libhermes",
```

Comment:
if you're putting hermes in a react-native location, you should put jsi in the same location since it's specific to that hermes lib

Reply:
good point